build/bakefiles/files.bkl

```diff
@@ -883,7 +883,6 @@ IMPORTANT: please read docs/tech/tn0016.txt before modifying this file!
     src/common/lboxcmn.cpp
     src/common/listctrlcmn.cpp
     src/common/markupparser.cpp
-    src/common/matrix.cpp
```

Comment:
This file isn't supposed to be modified directly, please update `build/files` and run `./build/upmake` instead.